GitHubApiWrapper/GitHubClient.cs

```diff
@@ -1,58 +1,20 @@
 ﻿using System;
-using System.Linq;
 using System.Net.Http;
 using System.Net.Http.Headers;
-using System.Threading.Tasks;
-using GitHubApiWrapper.Models;
-using GitHubApiWrapper.Models.CustomExceptions;
 
 namespace GitHubApiWrapper
 {
     public class GitHubClient
     {
-        private readonly HttpClient _client;
+        public readonly HttpClient Client;
```

Comment:
Exposing the `HttpClient` allows the developer to overwrite the authentication token, possibly causing annoying issues in the future.

Reply:
It's readonly, and has no set accessor

Reply:
It's readonly, so they shouldn't be able to set a new instance to it

Reply:
But the property `Client.DefaultRequestHeaders.Authorization` isn't and that may be a problem.

Reply:
I see. Good point. I can hide the underlying client and expose public methods instead.

Reply:
In my view, that would be good approach.